cowamm/kpi/competitors/cow_amm/cow_amm_kpi_4340428.sql

```diff
@@ -0,0 +1,24 @@
+-- Computes volume, tvl and APR for CoW AMM pools
+-- APR is measured as the fees earned per $ invested, over the last 24 hours, projected over 1 year
+-- Parameters:
+-- {{blockchain}}: The blockchain to query
+-- {{competitor_end_time}}: The end time of the time window (end_time - 1 day; end_time), defaults to now()
```

Comment:
I'm just realising that this value isn't really useful unless it's also populated from the query above. Otherwise you won't be able to call the data pipeline for a previous date.